test/unit/merge.js

```diff
@@ -1215,72 +1216,103 @@ describe('Merge', function() {
         .upToDate()
         .remoteId(dbBuilders.id())
         .create()
-      const orig = await builders
+      const file = await builders
         .metafile()
         .path('SRC/FILE')
-        .sides({ [this.side]: 2, [otherSide(this.side)]: 2 })
+        .upToDate()
         .remoteId(dbBuilders.id())
         .create()
-      const was = await builders
-        .metafile()
+      const file2 = builders
+        .metafile(file)
         .path('SRC/FILE2')
-        .moveFrom(orig)
-        .sides({ [this.side]: 3, [otherSide(this.side)]: 2 })
-        .remoteId(dbBuilders.id())
-        .create()
+        .build()
+      await this.merge.moveFileAsync(
```

Comment:
Maybe we could wrap the build+merge+get logic with some helper to make this test a bit shorter & reuse it in the following one too?

Either a plain function:

```js
const was = await builders.move(file, 'SRC/FILE2')
```

Or a builder class in case it makes sense:

```js
const was = await builders.move(file).to('SRC/FILE2').create()
```

Assuming `Merge` is injected into `Builders` together with `Pouch`:

```js
// replacing:
builders = new Builders({ pouch: this.pouch })
// with:
builders = new Builders(this)
```